frontend/src/index.js

```diff
@@ -0,0 +1,20 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+
+import reducer from './reducers'
+import './Components/Styles/index.scss';
+
+import App from './components/App';
+
+const createStoreWithMiddleware = applyMiddleware(thunk)(createStore);
+
+ReactDOM.render(
+    <Provider store={createStoreWithMiddleware(reducer)}>
```

Comment:
Instead of inlining statements into JSX markup, it's better to create a variable and use the variable.